### DISCUSSION TITLE
TABLES IN SCHEMA option, supported by v15

### DISCUSSION THREAD

**Discussion by roccho-dev** [electric-sql/pglite#253] [Q&A] [answered]
I'm looking for a simple way for pglite on the browser to sync with the server.

This option enables logical replication between databases.
Does this option, supported since postgres15, also support pglite?
I would also like to know what version of postgres pglite is based on.

**Comment by samwillis** [comment, marked as answer]
Hey @PorcoRosso85 

We haven't worked on Postgres logical replication yet. There is some complexity to overcome yet as it requires multiple processes.

We do however have an early work-in-progress sync plugin using our (ElectricSQL) sync engine: https://pglite.dev/docs/sync. This is going to be fleshed out more in the coming withs with more functionality.

**Reply by natew** [reply]
@samwillis curious how hard you think it would be to get postgres-native replication working with pglite. I'd be interested in contributing those changes, but would like to get an idea of the complexity/difficulty, as I'm very unfamiliar with postgres architecture in general.

**Comment by roccho-dev** [comment]
@samwillis 

Typescript / Elixir client can update also schema (not only data) of PGlite?
https://next.electric-sql.com/api/clients/typescript
